src/Symfony/Component/Asset/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/asset",
+    "type": "library",
+    "description": "Symfony Asset Component",
+    "keywords": [],
```

Comment:
it would be better to define some keywords to make it easier to find in the Packagist search